packages/react-button/lib/button.js

```diff
@@ -19,14 +20,17 @@ const Button = ({
 
 	let content = (
 		<React.Fragment>
-			{icon && "" !== icon && (
+			{icon && !placeIconRight && "" !== icon && (
 				<span className={"sui-icon-" + icon} aria-hidden="true" />
 			)}
 			{label}
+			{icon && placeIconRight && "" !== icon && (
+				<span className={"sui-icon-" + icon} aria-hidden="true" />
+			)}
 		</React.Fragment>
 	);
 
-	className = `sui-button${ className ? ' ' + className : '' }`;
+	className = `sui-button ${placeIconRight ? 'sui-button-icon-right' : ''} ${ className ? ' ' + className : '' }`;
```

Comment:
@Gowtham369 let's avoid unnecessary white space before/after a conditional class. Instead, add the class if you need it. As you can see in the following line:
```
${ className ? ' ' + className : '' }
```
Notice that before `className` I added an empty space between single quotes.